### ISSUE TITLE
Hotkey (function key) no longer works after update 1.7.2 -> 1.7.6

### ISSUE THREAD

**Issue by sp681** [moses-palmer/pynput#476]
**Description**
After updating pynput from version 1.7.2 to 1.7.6, the function key `<f8>` hotkey no longer works. Other hotkeys like `<ctrl>+q` work fine. 

**Platform and pynput version**
Windows 10, pynput 1.7.6

**To Reproduce**
```
from pynput import keyboard

def on_activate():
    print("Hotkey pressed")

def for_canonical(f):
    return lambda k: f(hotkeylistener.canonical(k))

hotkey_string = '<f8>'
hotkey = keyboard.HotKey(keyboard.HotKey.parse(hotkey_string), on_activate)
hotkeylistener = keyboard.Listener(
    on_press=for_canonical(hotkey.press),
    on_release=for_canonical(hotkey.release))
hotkeylistener.start()

while True:
    pass
```

**Comment by xageorges** [comment]
PR #466 fixes this issue.

**Comment by moses-palmer** [comment]
Thank you for your report, and thank you @xageorges for pointing out the PR!

I have merged a similar patch, and verified that hotkeys containing function keys are now handled correctly. The fix will be included in *pystray 1.7.7*.

**Comment by xageorges** [comment]
@moses-palmer: it is I who thank you for your library!
